### PR TITLE
Added check for directory

### PR DESCRIPTION
**Pull request by Diosa34**
Checking if user's file is not a directory

### CODE REVIEW

File: src/main/kotlin/commands/Functions.kt [hunk 1]
```diff
@@ -21,17 +21,27 @@ fun <T: Any> tryGet(field: String, t: Int, message: String, number: String.() ->
             field.number().let{ number ->
                 if(number == null){
                     println("Данные некорректны")
-                    println(message)
-                    println("Количество оставшихся попыток: ${t-i-1}")
+                    if (i != t-1) {
```
Comment (seductorAmadeus) [inline src/main/kotlin/commands/Functions.kt:24]: Please reduce duplicated code :)

Comment (Diosa34) [reply src/main/kotlin/commands/Functions.kt]: Done

File: src/main/java/com/github/Diosa34/DMS_HelloWorld/classes/Vehicle.java [hunk 2]
```diff
@@ -94,8 +94,8 @@ public FuelType getFuelType() {
     /**
      * A new ID is created that is one greater than the largest existing one.
      */
-    public static Integer idGenerator(){
-        Integer max_id = 0;
+    public static int idGenerator(){
+        int max_id = 0;
```
Comment (seductorAmadeus) [inline src/main/java/com/github/Diosa34/DMS_HelloWorld/classes/Vehicle.java:98]: Replace max_id with maxId

File: src/main/java/com/github/Diosa34/DMS_HelloWorld/parsing/CommandExecuter.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import com.github.Diosa34.DMS_HelloWorld.ApplicationPart;
+import com.github.Diosa34.DMS_HelloWorld.commands.Command;
+import com.github.Diosa34.DMS_HelloWorld.enums.InstanceCreator;
+import kotlin.Pair;
+import org.jetbrains.annotations.NotNull;
+
+
+public class CommandExecuter implements ApplicationPart {
+    private AbstractParser parser;
+    private Application applicationInstance;
```
Comment (seductorAmadeus) [inline src/main/java/com/github/Diosa34/DMS_HelloWorld/parsing/CommandExecuter.java:13]: Add final modifier here

File: src/main/java/com/github/Diosa34/DMS_HelloWorld/Application.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package com.github.Diosa34.DMS_HelloWorld;
+
+
+public class Application {
+    public String filepath;
+
+    public Application(String filepath){
+        this.filepath = filepath;
+    }
+}
```
Comment (seductorAmadeus) [inline src/main/java/com/github/Diosa34/DMS_HelloWorld/Application.java:10]: Please add javadoc

**Review by seductorAmadeus — CHANGES_REQUESTED** [review]

**Pull request merged**